qldbdriver/result.go

```diff
@@ -21,15 +21,17 @@ import (
 
 // Result is a cursor over a result set from a QLDB statement.
 type Result struct {
-	ctx          context.Context
-	communicator qldbService
-	txnID        *string
-	pageValues   []*qldbsession.ValueHolder
-	pageToken    *string
-	index        int
-	logger       *qldbLogger
-	ionBinary    []byte
-	err          error
+	ctx               context.Context
+	communicator      qldbService
+	txnID             *string
+	pageValues        []*qldbsession.ValueHolder
+	pageToken         *string
+	index             int
+	logger            *qldbLogger
+	ionBinary         []byte
+	consumedIOs       *IOUsage
```

Comment:
Can we put consumedIOs and timingInformation in a metrics or stats variable?

Reply:
Moved `consumedIOs` and `timingInformation` to a `statementStatistics` struct.

Reply:
Just call this `metrics`

Reply:
Updated to `metrics`.